libraries/AP_HAL/board/linux.h

```diff
@@ -332,6 +332,19 @@
 #define HAL_GPIO_C_LED_PIN        16
 #define HAL_GPIO_LED_ON           LOW
 #define HAL_GPIO_LED_OFF          HIGH
+#elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_EDGE
+#define HAL_BOARD_LOG_DIRECTORY "/var/APM/logs"
```

Comment:
we should stop doing this... /var/lib/ardupilot/logs would be the more FHS-compliant dir. Can you move to it?

Reply:
Yeah. Sure. Thanks!